cosmoz-viewinfo.js

```diff
@@ -105,64 +105,66 @@
 		listeners: {
 			'iron-resize': '_onResize'
 		},
-		_effectsChanged: function (newValue) {
+		_effectsChanged(newValue) {
 			this._notifyInstances({ effects: newValue });
 		},
 		/**
-		 * Loop over registered ViewInfoBehavior components and notify of changes<br/>
-		 * TODO: Don't reset the viewInfo property, but rather notify specific properties
+		* Loop over registered ViewInfoBehavior components and notify of changes<br/>
+ 	 * TODO: Don't reset the viewInfo property, but rather notify specific properties
+		 *
+		 * @param  {Object} delta object with changes
+		 * @returns {void}
 		 */
-		_notifyInstances: function (delta) {
-			viewInfoInstances.forEach(function (instance) {
+		_notifyInstances(delta) {
+			viewInfoInstances.forEach(instance => {
 				if (!instance) {
 					return;
 				}
-				Object.keys(delta).forEach(function (key) {
+				Object.keys(delta).forEach(key => {
 					instance.notifyPath('viewInfo.' + key, delta[key]);
 				});
 			});
 		},
 		/**
 		 * Called on `iron-resize`, throttles `viewinfo-resize` events
+		 * @returns {void}
 		 */
-		_onResize: function () {
-			var
-				update = this._updateViewSize(),
-				throttleFunction;
+		_onResize() {
+			const update = this._updateViewSize(),
+				throttleFunction = function () {
```

Comment:
Should be arrow function, especially with a .bind(this)
Or maybe rather an own method to avoid re-creating the function